game-core/src/main/java/games/strategy/debug/error/reporting/UserErrorReport.java

```diff
@@ -1,35 +1,39 @@
 package games.strategy.debug.error.reporting;
 
-import java.util.logging.LogRecord;
+import java.util.Optional;
 
+import javax.annotation.Nullable;
+
+import games.strategy.engine.framework.system.SystemProperties;
 import org.triplea.http.client.error.report.create.ErrorReport;
-import org.triplea.http.client.error.report.create.ErrorReportDetails;
 
 import games.strategy.engine.ClientContext;
-import lombok.AccessLevel;
 import lombok.Builder;
-import lombok.EqualsAndHashCode;
-import lombok.Getter;
-import lombok.ToString;
 
 /**
- * Represents the data a user has entered into a swing GUI.
+ * The user error report window can be seeded with information from the console or an error that
+ * impacted the user. The user is given a chance to enter description information of their own. This
+ * data object represents the bundle of that data after a user has entered data.
+ * <p>
+ * Or, a user can open a new bug report and simply enter a description and submit it.
+ * </p>
  */
 @Builder
-@EqualsAndHashCode
-@ToString
-@Getter(AccessLevel.PACKAGE)
 class UserErrorReport {
-  private final String title;
+
+  @Nullable
   private final String description;
-  private final LogRecord logRecord;
+  @Nullable
+  private final String errorData;
 
   ErrorReport toErrorReport() {
-    return new ErrorReport(ErrorReportDetails.builder()
-        .gameVersion(ClientContext.engineVersion().getExactVersion())
-        .logRecord(logRecord)
-        .title(title)
-        .description(description)
-        .build());
+    return ErrorReport.builder()
+        .operatingSystem(SystemProperties.getOperatingSystem())
+        .javaVersion(SystemProperties.getJavaVersion())
```

Comment:
Per the Javadocs, `java.version` is guaranteed to exist while `java.runtime.version` is not.  So I guess the main screen should be changed.  Out of scope for this PR.

Reply:
I do not think it is consistency that is at question, a user would have to be very keen to play a game for 2 hours, crash, create a bug report, then notice on a preview window or the created-issue that the java version did not match what they saw on the original launch screen and was missing a build number, but that would be really cool if someone did :smile: 

Very salient though is if we are concerned with java version build number and if that helps. I don't have context to know how conscious of a choice that was the main screen. For bug report I would lean on the side having more information rather than less, particularly since the value is for debugging/reporting and is not displayed to user unless they preview the 'raw' report. 

> Not sure if you want to do that in this PR or not.

I think if the question comes up for a reviewer, the answer is going to be in general no and a fast-follow PR should be done. How to do that well is a good question. 

I added: a63bd2d46 to address this. There is an API update included to abstract away the 'spec' vs 'runtime' detail, typically when we want 'java version' we are thinking `java --version` kind of output and want the more detailed value.

Reply:
Not sure if you saw my previous comment, but `java.runtime.version` is not guaranteed to exist on every JVM.  I can't find any official documentation on it.  We should probably play it safe and use `java.version`, as the [Javadocs](https://docs.oracle.com/javase/8/docs/api/java/lang/System.html#getProperties--) guarantee it will always have a value.  Otherwise, we'll need to deal with a possible `null` value.

Reply:
Fun. Good find :+1:

Reply:
Okay, so bouncing back to you to swap the `java.runtime.version` string for `java.version` in `SystemProperties#getJavaVersion()`.

Reply:
Apologies, I lost track of things a bit and thought an update already had been pushed.

That update is pushed now: e26cc1eb9